.gitignore

```diff
@@ -5,3 +5,15 @@ obj/
 *.cache
 GameDataParser/Resources
 Maple2Storage/Resources
+/DatabaseHandler/Character.cs
+/DatabaseHandler/Account.cs
```

Comment:
can we put these in a folder so we can just ignore the entire folder instead of each table's file individually?